src/version.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
 module.exports = {
-  defaultVersion: '1.20.2',
-  supportedVersions: ['1.7', '1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.5', '1.17.1', '1.18.2', '1.19', '1.19.2', '1.19.3', '1.19.4', '1.20', '1.20.1', '1.20.2']
+  defaultVersion: '1.20.3',
```

Comment:
Do you think it's useful to have 1.20.3 there ? What about testing only 1.20.4 ?

Reply:
Ah missed changing the defaultVersion to 1.20.4 when I added the 1.20.4 to the supportedVersions list.